packages/core/components/FileDetails/index.tsx

```diff
@@ -49,63 +49,107 @@ export default function FileDetails(props: FileDetails) {
+                        </div>
+                    </div>
+                    <Pagination
+                        className={classNames(styles.pagination, {
                             [styles.hidden]: windowState.state === WindowState.MINIMIZED,
                         })}
-                    >
-                        {thumbnailUriPath && (
-                            <div
-                                className={classNames(styles.fileThumbnailContainer, {
-                                    [styles.thumbnailDefault]:
-                                        windowState.state === WindowState.DEFAULT,
-                                    [styles.thumbnailMaximized]:
-                                        windowState.state === WindowState.MAXIMIZED,
-                                })}
-                            >
-                                <FileThumbnail
-                                    uri={`http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${thumbnailUriPath}`}
-                                />
+                    />
+                    <div className={styles.contentContainer}>
+                        <div
+                            className={classNames(styles.overflowContainer, {
+                                [styles.hidden]: windowState.state === WindowState.MINIMIZED,
+                            })}
+                        >
+                            {thumbnailUriPath && (
+                                <div
+                                    className={classNames(styles.fileThumbnailContainer, {
+                                        [styles.thumbnailDefault]:
+                                            windowState.state === WindowState.DEFAULT,
+                                        [styles.thumbnailMaximized]:
+                                            windowState.state === WindowState.MAXIMIZED,
+                                    })}
+                                >
+                                    <FileThumbnail
+                                        uri={`http://aics.corp.alleninstitute.org/labkey/fmsfiles/image${thumbnailUriPath}`}
+                                    />
+                                </div>
+                            )}
+                            <div className={styles.fileActions}>
+                                <Download fileDetails={fileDetails} />
                             </div>
-                        )}
-                        <div className={styles.fileActions}>
-                            <Download fileDetails={fileDetails} />
+                            <FileAnnotationList
+                                className={styles.annotationList}
+                                fileDetails={fileDetails}
+                                isLoading={isLoading}
+                            />
+                            <div className={styles.spacer} />
+                            <div className={styles.gradientTeaser} />
```

Comment:
I think something happened to this gradient teaser...not seeing it render anymore